seeds/userData.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
We'll probably want to have a user seed for each superhero seed, so that all the heroes have account details.